go/vt/vtgate/executor.go

```diff
@@ -246,7 +248,10 @@ func (e *Executor) Execute(ctx context.Context, mysqlCtx vtgateservice.MySQLConn
 
 	logStats.SaveEndTime()
 	e.queryLogger.Send(logStats)
+
+	err = errorTransform.TransformError(err)
```

Comment:
nitpick: maybe we should make the default errorTransformer handle the error truncation? not important, just thinking out loud

Reply:
I assumed that we'd always want to retain the truncation behavior even if other components transform the error. This would still be possible if we put the truncation in the default transformer, but we'd need to remember to wrap the default transformer rather than just adding the transformation.

I don't have strong opinions though so if you think it's better have the default transformation be truncation, let me know and I'll make the change 👍

Reply:
Nah, you're right.